db.sql

```diff
@@ -339,3 +339,34 @@ INSERT INTO `UserFavorite` VALUES
     (10, 9, 1);
 
 COMMIT;
+
+-- View for Searching by Ingredient --
+CREATE VIEW recipeIngredientSearch AS
+	SELECT recipe_id, title AS recipe_title, photo, ingredient_id, name AS ingredient_name
+    	FROM RecipeIngredient INNER JOIN Recipe ON recipe_id = Recipe.id
+        INNER JOIN Ingredient ON ingredient_id = Ingredient.id;
+
+-- View for Searching by Author --
+CREATE VIEW recipeAuthorSearch AS
+	SELECT recipe_id, title, photo, username AS author
+    	FROM UserRecipe INNER JOIN Recipe ON recipe_id = Recipe.id
+        INNER JOIN User ON user_id = User.id;
+
+-- Procedure to Update Search Views --
+DELIMITER $$
+CREATE DEFINER=`cs340_vaughanh`@`%` PROCEDURE `updateSearchViews`()
```

Comment:
This isn't a huge deal but we need to keep an eye out if we're using vaughanh or trotterj and update this accordingly